fe/fe-core/src/main/java/org/apache/doris/nereids/memo/Group.java

```diff
@@ -139,7 +165,28 @@ public List<GroupExpression> clearPhysicalExpressions() {
     }
 
     public double getCostLowerBound() {
-        return costLowerBound;
+        return -1D;
+    }
+
+    /**
+     * Get the lowest cost {@link org.apache.doris.nereids.trees.plans.physical.PhysicalPlan}
+     * which meeting the physical property constraints in this Group.
```

Comment:
meeting -> satisfies

Reply:
this is original comment, i do not mod it at all. all comment should re-check together